docs/src/examples/components/Label/Variations/LabelExampleOnIconClick.shorthand.tsx

```diff
@@ -1,31 +1,29 @@
 import React from 'react'
 import { Label } from '@stardust-ui/react'
 
-class LabelExampleIconShorthand extends React.Component<{}, { display: string }> {
-  constructor(props) {
-    super(props)
-    this.state = {
-      display: 'inline-block',
-    }
-  }
+class LabelExampleOnIconClickShorthand extends React.Component {
+  state = { hidden: false }
 
-  public hide = () => {
-    this.setState({ display: 'none' })
+  hide = () => {
+    this.setState({ hidden: true })
+    setTimeout(() => this.setState({ hidden: false }), 2000)
```

Comment:
There was previously no way to try this example again after "dismissing" the label.